core/arch/arm/include/sm/optee_smc.h

```diff
@@ -278,6 +278,9 @@
  */
 #define OPTEE_SMC_SEC_CAP_DYNAMIC_SHM		(1 << 2)
 
+/* Secure world supports Shared Memory with a NULL Reference */
+#define OPTEE_SMC_SEC_CAP_MEMREF_NULL		(1 << 3)
```

Comment:
We have `OPTEE_SMC_SEC_CAP_VIRTUALIZATION` on that bit, so this must be change to `1 << 4`.

Reply:
done